carta/image.py

```diff
@@ -477,17 +477,24 @@ def set_scaling(self, scaling, alpha=None, gamma=None, min=None, max=None):
             The alpha value (only applicable to ``LOG`` and ``POWER`` scaling types).
         gamma : {2}
             The gamma value (only applicable to the ``GAMMA`` scaling type).
-        min : {3}
-            The minimum of the scale. Only used if both *min* and *max* are set.
-        max : {4}
-            The maximum of the scale. Only used if both *min* and *max* are set.
+        clip : {3}
+            The scale clip. In percentage. Can not be used witn *min* and *max*.
+        min : {4}
+            The minimum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
+        max : {5}
+            The maximum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
         """
         self.call_action("renderConfig.setScaling", scaling)
         if scaling in (Scaling.LOG, Scaling.POWER) and alpha is not None:
             self.call_action("renderConfig.setAlpha", alpha)
         elif scaling == Scaling.GAMMA and gamma is not None:
             self.call_action("renderConfig.setGamma", gamma)
-        if min is not None and max is not None:
+        if clip is not None:
+            min = "None"
+            max = "None"
+            self.call_action("renderConfig.setPercentileRank", clip)
+        elif min is not None and max is not None:
+            clip = "None"
             self.call_action("renderConfig.setCustomScale", min, max)
```

Comment:
1. You are not actually using these `"None"` strings after creating them, because neither block in the conditional is calling the other action.
2. It is unnecessary for you to unset these values in the frontend, because the frontend functions already do that. `setPercentileRank` automatically sets the clip min and max, and vice versa. You should remove these three lines entirely.

Reply:
I remove these three lines are removed in 7b81303.